fe_repo/src/functions/api.tsx

```diff
@@ -25,21 +64,24 @@ export function sendMessage(message: string) {
   }
 }
 
-export function analyze(job_description: string) {
+export function analyze(job_description: string, msg: Message, refreshMessages: () => void) {
   console.log("analyzing", job_description);
   const resp = {
     response: 'An error occurred, please try again later.',
   }
-  axios.post('/api/analyze', {job_description: job_description, user_id: getUserId()},
-    {headers: {'Content-Type': 'application/json'}})
-    .then(response => resp.response = response.data.result)
+  axios.post<AnalzyeResponse>(uri + '/resume_evaluate', {job_description: job_description, user_id: getUserId()},
+    {headers: {'Content-Type': 'multipart/form-data'}})
+    .then(response => {
+      msg.text = buildAnalysis(response.data);
```

Comment:
Call the `setMessages` here, and then the React will rerender the related component.